### PR TITLE
Update to airbase 119 & JUnit 5 and add support for JDK 17

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

**Comment by wendigo** [comment]
Cc @findepi @martint @electrum

File: tempto-kafka/pom.xml [hunk 1]
```diff
@@ -42,4 +42,15 @@
             <version>2.4.1</version>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <failIfNoTests>false</failIfNoTests>
```
Comment (findepi) [inline tempto-kafka/pom.xml]: Why would you want to do that?
If there are no tests, we should exclude surefire run.

File: tempto-examples/docker/docker-compose.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   hadoop-master:
     hostname: hadoop-master
-    image: 'ghcr.io/trinodb/testing/hdp2.6-hive:38'
+    image: 'ghcr.io/trinodb/testing/hdp2.6-hive:53'
```
Comment (findepi) [inline tempto-examples/docker/docker-compose.yml]: This, and jsch update are simplistic. Can you extract them to a separate PR for quick merge?

File: tempto-kafka/pom.xml (hunk 1 above)
Comment (wendigo) [reply tempto-kafka/pom.xml]: There are no tests in these modules but after airbase update surefire will fail on lack of tests

Comment (findepi) [reply tempto-kafka/pom.xml]: add this to properties instead:
```
 <!-- there are no tests -->
 <maven.test.skip>true</maven.test.skip>
```

Comment (electrum) [reply tempto-kafka/pom.xml]: Airbase 117 will have a property for this

File: pom.xml [hunk 3]
```diff
@@ -198,4 +198,18 @@
             </dependency>
         </dependencies>
     </dependencyManagement>
+
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <!-- TODO: Remove version override when Jacoco is updated in airbase -->
```
Comment (electrum) [inline pom.xml]: Is there a PR?

Comment (wendigo) [reply pom.xml]: I'll open it

File: tempto-kafka/pom.xml (hunk 1 above)
Comment (findepi) [reply tempto-kafka/pom.xml]: Seems you can use `maven.test.skip` for now, but need to switch to `air.test.skip` after airbase 117, as airbase upgrade will render `maven.test.skip` ineffective. (This is a breaking change in airbase.)

File: pom.xml (hunk 3 above)
Comment (wendigo) [reply pom.xml]: Nvm, I've seen that you've bumped it. I'll move to airbase 117

File: tempto-examples/docker/docker-compose.yml (hunk 2 above)
Comment (wendigo) [reply tempto-examples/docker/docker-compose.yml]: https://github.com/trinodb/tempto/pull/91  https://github.com/trinodb/tempto/pull/92

File: tempto-kafka/pom.xml (hunk 1 above)
Comment (wendigo) [reply tempto-kafka/pom.xml]: I've moved to airbase 117 to use `air.test.skip`

**Comment by wendigo** [comment]
@findepi @electrum please review now :) It's rebased on top of the extracted PRs

File: tempto-examples/pom.xml [hunk 4]
```diff
@@ -14,6 +14,7 @@
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
         <main-class>io.trino.tempto.examples.TemptoExamples</main-class>
+        <air.test.skip>true</air.test.skip>
```
Comment (findepi) [inline tempto-examples/pom.xml]: add `<!-- there are no tests in this module -->`

Comment (wendigo) [reply tempto-examples/pom.xml]: done

**Comment by wendigo** [comment]
PTAL @electrum @findepi

**Comment by electrum** [comment]
Sorry, please update to Airbase 119, as 117 breaks `skipTests`.

**Comment by wendigo** [comment]
@electrum updated to 119

**Review by kokosing — APPROVED** [review]
